sdk/src/types/block/protocol.rs

```diff
@@ -150,3 +155,12 @@ pub fn protocol_parameters() -> ProtocolParameters {
     )
     .unwrap()
 }
+
+impl_id!(
+    pub ProtocolParametersHash,
+    32,
+    "The hash of the protocol parameters for the Highest Supported Version."
```

Comment:
```suggestion
    "The hash of the protocol parameters."
```
That sentence is only relevant in the context of the TIP